chapter2.ml

```diff
@@ -100,6 +100,13 @@ let close_enough a b =
 let _ = assert (close_enough (area 1.0) Float.pi)
 let _ = assert (close_enough (area (Float.sqrt (1. /. Float.pi))) 1.)
 
+(********************************************************************
+ * exercise: RMS
+ ********************************************************************)
+
+let rms x y = sqrt((x *. x +. y *. y) /. 2.)
+(* assert (abs_float (rms 2. 2. -. 2.) < 1e-5) *)
```

Comment:
May be `let _ = assert (close_enough 2. (rms 2. 2.))` looks more familiar, as _more fun_ has taken same approach.